platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -325,13 +343,18 @@ public static Address buildAddress(@NonNull final RosterEntry entry) {
 
     /**
      * Build an AddressBook object out of a given Roster object.
+     * Returns null if the input roster is null.
      * @deprecated To be removed once AddressBook to Roster refactoring is complete.
      * @param roster a Roster
      * @return an AddressBook
      */
     @Deprecated(forRemoval = true)
-    @NonNull
+    @Nullable
     public static AddressBook buildAddressBook(@NonNull final Roster roster) {
+        if (roster == null) {
```

Comment:
The interface marking indicates the input can't be null.  Should the interface be changed to `@Nullable`?

Reply:
Good catch. Done.